src/components/Snippet/FileManager.vue

```diff
@@ -246,15 +247,18 @@ v-card.girder-data-browser-snippet
       offset-y,
       dark)
     v-list(dense)
-      v-list-item(@click="showAccessControlDialog=true")
+      v-list-item(@click="showAccessControlDialog=true", :disabled="!hasAccessPermission")
         v-list-item-title Access control
   v-dialog(
       v-model="showAccessControlDialog",
       max-width="700px",
       persistent,
+      eager,
       scrollable)
     girder-access-control(
+        v-if="actOnItem",
         :model="actOnItem",
+        :has-permission.sync="hasAccessPermission",
```

Comment:
Wont' this cause the dialog to flash while the request fires?  Like won't it open quickly then close again?

Reply:
Hm, it is the disability of the list item that will change. It will flash from disabled to enabled but it's relatively mild.